client/components/web-preview/index.jsx

```diff
@@ -160,7 +160,11 @@ export class WebPreview extends Component {
 	shouldRenderIframe() {
 		// Don't preload iframe on mobile devices as bandwidth is typically more limited and
 		// the preview causes weird issues
-		return ! this._isMobile || this.props.showPreview;
+		if ( this._isMobile && ! this.props.showPreview ) {
```

Comment:
Couldn't this function be simplified to just `return this.props.showPreview` ? I don't see how the `this._isMobile` condition checks for anything not satisfied by that.